golem-shard-manager/src/shard_management.rs

```diff
@@ -129,63 +124,67 @@ impl ShardManagement {
             change.notified().await;
 
             let (new_pods, removed_pods) = updates.lock().await.reset();
-            debug!("Shard management loop woken up by change; new pods: {new_pods:?}, removed pods: {removed_pods:?}");
-
-            // Getting a write lock while the rebalance plan is calculated and got persisted (but NOT applied)
-            let mut current_routing_table = routing_table.write().await;
-
-            for pod in removed_pods {
-                current_routing_table.remove_pod(&pod);
-            }
+            debug!(
+                new_pods=?new_pods,
+                removed_pods=?removed_pods,
```

Comment:
here should be possible to use Display instead of Debug for arguments for new_pods and removed_pods

Reply:
https://github.com/golemcloud/golem/pull/690/commits/0a86347c73d43333b0b3c22abe0447b3fe41bebe